docs/release_notes/july2023.md

```diff
@@ -46,11 +104,11 @@ Previously, users could only connect a cluster to a single Pipeline, limiting th
 With this enhancement, users can now attach clusters to different Pipelines in different tabs, enabling concurrent connections without interference.
```

Comment:
Do we now support having Prophecy open in multiple browser tabs?

Reply:
Yes we do